data/repos.yml

```diff
@@ -361,6 +452,12 @@
   sentry_url: false
   dashboard_url: false
 
+# Not sure if prototype (in which case, archive) or actively used (in which case, add)
+# Check with Duncan Garmonsway
+- repo_name: govuk-bigquery-analytics
```

Comment:
@nacnudus would you mind checking this? I'll tag you in a few more below too 👀

Reply:
Thanks, this one is for Data Services in Product & Technology.  I've remove the govuk tag.

Reply:
Thank you!